src/components/App.js

```diff
@@ -1,299 +0,0 @@
-  writeUserInfoDetails (coordArra) {
-    firebase.database().ref('users/' + this.state.user.uid + '/userDetails').set({
-      coord: coord
-    });
-  }
-
-  // here we are getting the user coord from firebase database
-  getUserInfoDetails () {
-    console.log('<<<<<<<<<<<<<<<<<<<')
-    const thisVal = this;
-
-    firebase.database().ref('users/' + this.state.user.uid ).on('value', function(snapshot) {
-      //console.log('snapshot:- ' + snapshot.val().coord );
-      //const anotherSelf = this;
-      if (snapshot.val()) {  
-        thisVal.setState({
-          dbCoordsNow: snapshot.val().coord,
-        });
-      }
-      console.log(thisVal.state.dbCoordsNow, '>>>>>>>>')
-    });
-  }
-
-  componentDidMount() {
-    this.timerId = setInterval(() => this.tick(), 3000)
-
-    // this.writeUserData(this.state.coordPosNow)
-
-    // if(this.state.user){  
-    //   firebase.database().ref('users/' + this.state.user.uid).set({
-    //     coord: coord
-    //   });
-    // }
-
-
-    // setInterval( () => {
-
-    //   var database = firebase.database()
-    //   var ref = database.ref('users/' + this.state.user.uid + '/run' )
-    //   ref.set({coord: coord});
-    // }, 1000
-    // )
-
-
-
-    // if (this.state.user && coord) {
-    // }
-
-// function writeUserData(userId, name, email, imageUrl) {
-//   firebase.database().ref('users/' + 'userId').set({
-//     username: 'name',
-//     email: 'email',
-//     profile_picture : imageUrl
-//   });
-// }
-
-    var val = this;
-    firebase.auth().onAuthStateChanged(function(user) {
-      if (user) {
-        // User is signed in.
-        console.log('*************');
-        (function () {
-          val.setState({
-            user: user
-          })
-        })();
-        
-        // console.log(this.state.user)
-      } else {
-        // No user is signed in.
-        console.log('===========') 
-        val.setState({
-          user: null
-        })
-      }
-    });
-
-    this.state.user ?
-    firebase.database().ref('users/' + this.state.user.uid ).on('value', function(snapshot) {
-      //console.log('snapshot:- ' + snapshot.val().coord );
-      if (snapshot.val()) {  
-        this.setState({
-          dbCoordsNow: snapshot.val().coord,
-        });
-      }
-      console.log(this.state.dbCoordsNow, '>>>>>>>>')
-    })
-    : null
-
-//     var database = firebase.database()
-// var ref = database.ref('users')
-// ref.set({username: 'Bangash'});
-  }
-
-  // clearing the setInterval Id's so that we dont have douplication of tasks being performed
-  compnoentWillMount(){
-    clearInterval(this.timerId);
-    clearInterval(this.dbtimerId);
-    clearInterval(this.getDbtimerId);
-  }
-
-  render () {    
-    // const condition = this.state.coords ? this.state.coords[this.state.coords.length -1] : 'false';
-    
-    // checking if user is looged in.
-    const showNameIfLoggedin = this.state.user ? this.state.user: false;
```

Comment:
`showNameIfLoggedin` isn't really needed. Delete it, and clean up any references.